packages/logger/src/register-logger-to-container.ts

```diff
@@ -38,11 +25,34 @@ export function registerLoggerToContainer(container: Container): void {
             return new GlobalLogger([appInsightsLoggerClient, consoleLoggerClient], context.container.get(loggerTypes.Process));
         })
         .inSingletonScope();
+}
+
+export function registerContextAwareLoggerToContainer(container: Container): void {
+    if (!container.isBound(Logger)) {
+        registerLoggerDependenciesToContainer(container);
+    }
 
-    container.bind(ContextAwareLogger).toDynamicValue(context => {
+    container.bind(Logger).toDynamicValue(context => {
         const appInsightsLoggerClient = context.container.get(ContextAwareAppInsightsLoggerClient);
         const consoleLoggerClient = context.container.get(ContextAwareConsoleLoggerClient);
 
         return new ContextAwareLogger([appInsightsLoggerClient, consoleLoggerClient], context.container.get(loggerTypes.Process));
     });
 }
+
+function registerLoggerDependenciesToContainer(container: Container): void {
+    container.bind(loggerTypes.AppInsights).toConstantValue(appInsights);
+    container.bind(loggerTypes.Process).toConstantValue(process);
+    container
+        .bind(AppInsightsLoggerClient)
+        .toSelf()
+        .inSingletonScope();
+    container
+        .bind(ConsoleLoggerClient)
+        .toSelf()
+        .inSingletonScope();
```

Comment:
This is context related bind for `registerGlobalLoggerToContainer()`

Reply:
Since it uses inSingletonScope there will be one global object for AppInsightsLoggerClient and ConsoleLoggerClient

Reply:
Right, but this is not a part of  `registerLoggerDependenciesToContainer()` and not need to be called from `registerContextAwareLoggerToContainer()`. The `registerLoggerDependenciesToContainer()` is intended to be common for both registrations